share/rpcauth/rpcauth.py

```diff
@@ -38,9 +39,14 @@ def main():
     salt = generate_salt(16)
     password_hmac = password_to_hmac(salt, args.password)
 
-    print('String to be appended to bitcoin.conf:')
-    print('rpcauth={0}:{1}${2}'.format(args.username, salt, password_hmac))
-    print('Your password:\n{0}'.format(args.password))
+    if args.output:
+        file = open(args.output, "x")
```

Comment:
Mode should probably be `'a'`

Reply:
I was thinking it should fail if the file already exists.

Reply:
Then how would you build a file with many lines?